src/pmdas/hdb/pmdahdb.1

```diff
@@ -0,0 +1,87 @@
+'\"macro stdmacro
+.\"
+.\" Copyright (c) 2021 Red Hat.
+.\"
+.\" This program is free software: you can redistribute it and/or modify it under
+.\" the terms of the GNU General Public License as published by the Free Software
+.\" Foundation, either version 3 of the License, or (at your option) any later
+.\" version.
+.\"
+.\" This program is distributed in the hope that it will be useful, but WITHOUT
+.\" ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+.\" FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+.\"
+.\" See <http://www.gnu.org/licenses/> for copy of the GNU General Public License.
+.\"
+.\"
+.TH PMDAHDB 1 "PCP" "Performance Co-Pilot"
+.SH NAME
+\f3pmdahdb\f1 \- SAP HANA pmda
+.SH DESCRIPTION
+\f3pmdahdb\f1 is a Performance Metrics Domain Agent (PMDA) which exports metric
+values about an SAP HANA (\fBH\fRigh-performance \fBAN\fRalytic \fBA\fRppliance) database,
+a multi-model database that stores data in its memory instead of keeping it on a disk.
+.PP
+This pmda can be used in scale-out scenarios. All metrics are Host and Port aware.
+Host and Port are included in the instance names (<host>.<port>.<instance>).
+.PP
+The HDB configuration is maintened with pmdahdb.conf. Curently there is only one hdb
```

Comment:
pmdahdb.conf -> newline, ".IR pmdandb.conf ."

The description of configuring things could be moved to INSTALLATION section.

Some typos here:  "maintained" and "Currently".

Mixing lower and upper base "HDB" here too, let's be consistent.